lib/ansible/modules/monitoring/nagios.py

```diff
@@ -261,6 +273,7 @@ def main():
             comment=dict(default='Scheduling downtime'),
             host=dict(required=False, default=None),
             servicegroup=dict(required=False, default=None),
+            start=dict(default=int(time.time())),
```

Comment:
I think it's better to set `start` to `None` by default. In this case `_fmt_dt_str()` with initialize it to `now()` anyway. And this avoid the case where the same variable can be initialized at two different locations in the code.